roles/openshift_named_certificates/tasks/main.yml

```
@@ -98,5 +99,5 @@
       owner: root
       group: root
   when:
-  - not ca_bundle_result
+  - ca_bundle_result.results[0].rc != 0
```

You will need to check if the item is defined first.
```suggestion
  - ca_bundle_result.results[0].rc is defined
  - ca_bundle_result.results[0].rc != 0
```
Is checking only the first item acceptable?  You could check all the items with something like:
```
when:
  - ca_bundle_result.results[0].rc is defined
  - 1 in (ca_bundle_result.results | map(attribute='rc') | list)
```

---

yeah, I know that I can check it, however, I wonder why the normal `ca_bundle_result.rc != 0` doesn't work. I see that it is used in the other playbooks, but for this doesn't work. I am registering the variable through `register: ca_bundle_result`.
Any idea how it can be changed? If not I can add the `defined` check.

---

`ca_bundle_result.rc` does not work here because the task registering in `ca_bundle_result` is looping with_items creating a list.  You must either process the list or choose an individual item.  You must add `is defined` check because the `results` may not contain the `rc` if the task is skipped.